mod/unfollow.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+use Friendica\App;
+use Friendica\Core\System;
+
+require_once 'include/probe.php';
+require_once 'include/follow.php';
+require_once 'include/Contact.php';
+require_once 'include/contact_selectors.php';
+
+function unfollow_content(App $a) {
+
+	if (! local_user()) {
+		notice(t('Permission denied.') . EOL);
+		goaway($_SESSION['return_url']);
+		// NOTREACHED
+	}
+
+	$uid = local_user();
+	$url = notags(trim($_REQUEST['url']));
+
+	$submit = t('Submit Request');
+
+	$condition = array("`uid` = ? AND `rel` = ? AND (`nurl` = ? OR `alias` = ? OR `alias` = ?) AND `network` != ?",
+			local_user(), CONTACT_IS_FRIEND, normalise_link($url),
+			normalise_link($url), $url, NETWORK_STATUSNET);
+	$contact = dba::select('contact', array('url', 'network', 'addr', 'name'), $condition, array('limit' => 1));
+
+	if (!dbm::is_result($contact)) {
+		notice(t("You aren't a friend of this contact.").EOL);
+		$submit = "";
+		// NOTREACHED
+	}
+
+	if (!in_array($contact['network'], array(NETWORK_DIASPORA, NETWORK_OSTATUS))) {
+		notice(t("Unfollowing is currently not supported by your network.").EOL);
+		$submit = "";
+		// NOTREACHED
+	}
+
+	$request = System::baseUrl()."/unfollow";
+	$tpl = get_markup_template('auto_request.tpl');
+
+	$r = q("SELECT `url` FROM `contact` WHERE `uid` = %d AND `self` LIMIT 1", intval($uid));
+
+	if (!$r) {
+		notice(t('Permission denied.') . EOL);
+		goaway($_SESSION['return_url']);
+		// NOTREACHED
+	}
+
+	$myaddr = $r[0]["url"];
+
+	// Makes the connection request for friendica contacts easier
+	$_SESSION["fastlane"] = $contact["url"];
+
+	$header = $contact["name"];
+
+	if ($contact["addr"] != "") {
+		$header .= " <".$contact["addr"].">";
+	}
+
+	$header = t("Disconnect/Unfollow");
```

Comment:
I guess this should be rather `$header .=`. Otherwise the content of `$header` will be overwritten

Reply:
Looks illogical ;-)